### ISSUE TITLE
Better environment names

### ISSUE THREAD

**Issue by aaronadamsCA** [cloudflare/pages-action#54]
TLDR, environment name should be **project-name (Production)** or **project-name (Preview)**. This aligns much better with environment functionality in both Cloudflare Pages and GitHub, and results in a much better UX.

---

This action now includes monorepo support, so I can run it for two Cloudflare projects at once. Currently I can't tell these deployments apart, or configure these environments individually in GitHub.

Worse, `environmentName` unnecessarily includes the branch name. This results in a new GitHub environment for every branch ([you can see how messy this gets here](https://github.com/cloudflare/pages-action/deployments)). This seems incorrect; in Cloudflare Pages there are only two environments, Preview and Production. I believe this action should follow the same logic and create only two GitHub environments per Cloudflare Pages project.

So, I'd like to propose updating the calculated environment name to always follow the same template: **project-name (Production)** or **project-name (Preview)**.

I've done this locally, and as you can see the behaviour is much clearer, and mirrors the Cloudflare dashboard.

---

![image](https://user-images.githubusercontent.com/1505561/214843869-3aded777-b624-40c2-a0cf-9a473f7ea063.png)

---

![image](https://user-images.githubusercontent.com/1505561/214843738-b3aa7f03-571a-473f-912f-7bfaf73632ff.png)
